stable/postgresql/values.yaml

```diff
@@ -187,6 +187,11 @@ service:
   ##
   # loadBalancerIP:
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
+  #  - 10.10.10.0/24
```

Comment:
Can you add a line only with ## between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

Reply:
ok added